### PR TITLE
Report to pdf

### PR DESCRIPTION
**Pull request by robzan8**
Trik, vedi un po' se ti piace o se ci sono modifiche da fare!

### CODE REVIEW

File: rollup-globals.bzl [hunk 1]
```diff
@@ -72,6 +72,7 @@ ROLLUP_GLOBALS = {
     "moment": "moment",
     "ngx-color-picker": "ngxColorPicker",
     "numbro": "numbro",
+    "pdfmake": "pdfmake",
```
Comment (trik) [inline rollup-globals.bzl:75]: this one should be
`"pdfmake/build/pdfmake": "pdfmake.build.pdfmake"`

File: src/core/reports/report-to-pdf/load-report-images.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Advanced JSON forms (ajf).
+ * If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {
```
Comment (trik) [inline src/core/reports/report-to-pdf/load-report-images.ts:23]: you can't import by module name inside the same module

File: src/core/reports/report-to-pdf/report-to-pdf.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Advanced JSON forms (ajf).
+ * If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {
```
Comment (trik) [inline src/core/reports/report-to-pdf/report-to-pdf.ts:23]: you can't import by module name inside the same module

**Review by trik — CHANGES_REQUESTED** [review]
please squash to a single commit

**Comment by robzan8** [comment]
Updated with requested changes!

**Comment by robzan8** [comment]
Resolved conflicts with master

**Review by trik — APPROVED** [review]

**Pull request merged**